### ISSUE TITLE
v-model change not updating calendar

### ISSUE THREAD

**Issue by andreliem** [jrainlau/vue-flatpickr#19]
Hi,

I'm using `v-model=currDate` to set the date of the calendar. But it seems like if the value of currDate changes the calendar will not pick this up.

Is there a proper way of having the value of a calendar changed?

**Comment by lmiller1990** [comment]
I do not think you can use this with v-model in it's current state. Consider a local value in your data () { } function, and then on @input, update the local value. 

To be used with v-model, two things are needed:

- accept a `value` prop
- emit an `input` event with the new value (not done)

For more information see [https://vuejs.org/v2/guide/components.html#Form-Input-Components-using-Custom-Events](url)

I would also like to use this with v-model, it would be a lot nicer. I'll look at doing that and make a PR, but it probably won't be until sometime after the new year. 

Is there any reason you want to to use v-model? You can get a similar behaviour using update and a local variable - if you are having trouble with this I can help if you post a jsfiddle of what you have so far.

**Comment by solidevolution** [comment]
Other question: Isn't there a possibility to do it with v-model? 

I have an big object with over 150 fields and 8 of them are dates. I don't want to duplicate these variables only because there are no good vuejs date pickers in the open source world ...

**Comment by aldoromo88** [comment]
I've made that changes in this [pull request](https://github.com/jrainlau/vue-flatpickr/pull/20)  please see changes on vue-flatpickr-base.vue file if you prefer modify it by yourself

**Comment by andreliem** [comment]
Thanks for updates @lmiller1990 @aldoromo88 ... I've tried the latest PR and it's still not quite working for me. 

All I'm doing is setting a value from the `v-model` to a data variable like `startDate`. 

If I change the value of `startDate` to another date in the Vue Console the calendar will not reflect this date visually, it only reflects the last date I picked with through the UI (mouse cursor). So basically the 2 way binding isn't reflected on the UI... although I can see it's being set internally by looking at the `value` field.

The reason I want this functionality is because I use the `flatpickr` calendar in a scheduling program which has a series of dates. When a user selects a date, the flatpickr calendar should be able to reflect the correct date via the `v-model` value passed to it on load. It only seems to do this the first time but changes to the `v-model` are not being picked up.

Hope that makes sense?
